.circleci/config.yml

```diff
@@ -79,56 +83,116 @@ aliases:
       - ~/watchman
     key: v1-watchman-v4.9.0
 
+  - &install-node-dependencies
+    |
+      npm config set spin=false
+      npm config set progress=false
+      npm install --no-package-lock
```

Comment:
We can consolidate this into a single command:

`npm install --no-package-lock --no-spin --no-progress`

(Using the separate `npm config` commands is good when changing the preferences globally, but we're mostly running `npm install` on separate images where the preference won't persist)